packages/eslint-config-airbnb/rules/es6.js

```diff
@@ -57,6 +53,12 @@ module.exports = {
     // http://eslint.org/docs/rules/prefer-template
     'prefer-template': 2,
     // disallow generator functions that do not have yield
-    'require-yield': 0
+    'require-yield': 0,
+    // import sorting
+    // http://eslint.org/docs/rules/sort-imports
+    'sort-imports': 0,
+    // enforce spacing around the * in yield* expressions
+    // http://eslint.org/docs/rules/yield-star-spacing
+    'yield-star-spacing': 0
```

Comment:
altho we don't use generators yet, just like `function* foo` is the correct spacing, so too would `yield* foo` be - i think it'd be fine to turn this on too :-)